src/coprocessor/dag/executor/aggregation.rs

```diff
@@ -143,7 +143,9 @@ impl<'a> Executor for AggregationExecutor<'a> {
         let value_size = group_key.len() + approximate_size(&aggr_cols, false);
         let mut value = Vec::with_capacity(value_size);
         box_try!(value.encode(aggr_cols.as_slice(), false));
-        value.extend_from_slice(group_key);
+        if !self.group_by.is_empty() {
+            value.extend_from_slice(group_key);
```

Comment:
It seems different with `aggregation` in `select` ?

Reply:
When there is no group by column, group_key is redundant, and since we no longer transfer rowMeta, redundant value shouldn't be sent to tidb.